crates/exporter/src/fetch_hash.rs

```diff
@@ -81,6 +81,8 @@ pub async fn fetch_hash(
 
 pub async fn compute_hash(asset_url: &str) -> anyhow::Result<String> {
     let sha256 = tokio::process::Command::new("nix-prefetch-url")
+        .arg("--name")
+        .arg("prefetched-asset.zip") // Ensure filename is valid for Open VSX URLs containing @
```

Comment:
what is this?

Reply:
If `nix-prefetch-url` is passed a URL containing an `@` or another character that's not allowed by `libstore` in the last path component, then it'll fail to write it e.g.

```sh
 nix-prefetch-url 'https://open-vsx.org/api/rust-lang/rust-analyzer/linux-x64/0.3.2593/file/rust-lang.rust-analyzer-0.3.2593@linux-x64.vsix'
```

```
error: path 'hfl905vm9097fxrvkk1b3f0mq7id8jn6-rust-lang.rust-analyzer-0.3.2593@linux-x64.vsix' is not a valid store path: name 'rust-lang.rust-analyzer-0.3.2593@linux-x64.vsix' contains illegal character '@'
```

By setting `--name` to another valid value it appears to work around the limitation, although it might be better if `nix-prefetch-url` handled the replacement for us.